sdk/keyvault/README.md

```diff
@@ -11,7 +11,7 @@ This project provides client libraries in JavaScript that makes it easy to consu
 - @azure/keyvault-keys [Package (npm)](https://www.npmjs.com/package/@azure/keyvault-keys)
 - @azure/keyvault-secrets [Package (npm)](https://www.npmjs.com/package/@azure/keyvault-secrets)
 - @azure/keyvault-certificates [Package (npm)](https://www.npmjs.com/package/@azure/keyvault-certificates)
-- [API Reference documentation](https://docs.microsoft.com/javascript/api/overview/azure/key-vault)
+- [API Reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/azure/key-vault-index?view=azure-node-latest)
```

Comment:
I recall the guidance was to not use the locale in the urls so that the right locale gets used based on the user's settings/location

Reply:
Also, we shouldnt need to add the query parameter for `view=azure-node-latest`. The link should get re-directed correctly

Reply:
```suggestion
- [API Reference documentation](https://docs.microsoft.com/javascript/api/overview/azure/key-vault-index)
```

Reply:
Removed it. Thanks for letting me know

Reply:
I will make similar updates in my other PR too.